rootfs/etc/nginx/lua/test/balancer/sticky_test.lua

```diff
@@ -0,0 +1,149 @@
+local sticky = require("balancer.sticky")
+local cookie = require("resty.cookie")
+local util = require("util")
+
+function get_mocked_ngx_env()
```

Comment:
can you do the mocking similar to what's done at https://github.com/kubernetes/ingress-nginx/blob/master/rootfs/etc/nginx/lua/test/util_test.lua#L6 ?

eventually I'd like to extract that function into a test helper module, therefore it'd be better to be repeat consistently at least for now.